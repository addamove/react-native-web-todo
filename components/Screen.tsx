import React from "react";
import { View, StyleSheet } from "react-native";

export const Screen: React.FunctionComponent<{}> = ({ children }) => {
  return <View style={styles.content}>{children}</View>;
};

const styles = StyleSheet.create({
  content: {
    padding: "0.5rem",
  },
});
