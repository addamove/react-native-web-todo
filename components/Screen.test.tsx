import React from "react";
import { render } from "@testing-library/react-native";
import { Screen } from "./Screen";
import { Text } from "react-native-elements";

test("Screen render correctly", () => {
  const { toJSON } = render(
    <Screen>
      <Text>Test</Text>
    </Screen>
  );

  expect(toJSON()).toMatchSnapshot();
});
