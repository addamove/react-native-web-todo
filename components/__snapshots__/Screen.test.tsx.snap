// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen render correctly 1`] = `
<View
  style={
    Object {
      "padding": "0.5rem",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#242424",
      }
    }
  >
    Test
  </Text>
</View>
`;
